docs/encrypt_decryt_configuration.md

```diff
@@ -0,0 +1,98 @@
+# Encrypt Configuration
+
+When you work in multiple environments: local, dev, testing, production... you must set critical configuration in your
+variables, like:
+
+config.yml, for local propose:
+```yaml
+pyms:
+  config:
+    DEBUG: true
+    TESTING: true
+    APPLICATION_ROOT : ""
+    SECRET_KEY: "gjr39dkjn344_!67#"
+    SQLALCHEMY_DATABASE_URI: mysql+mysqlconnector://user_of_db:user_of_db@localhost/my_schema
+```
+
+config_pro.yml, for production environment:
+```yaml
+pyms:
+  config:
+    DEBUG: true
+    TESTING: true
+    APPLICATION_ROOT : ""
+    SECRET_KEY: "gjr39dkjn344_!67#"
+    SQLALCHEMY_DATABASE_URI: mysql+mysqlconnector://important_user:****@localhost/my_schema
+```
+
+You can move this file to a [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/), 
+use [Vault](https://learn.hashicorp.com/vault) or encrypt the configuration with [AWS KMS](https://aws.amazon.com/en/kms/)
+ or [Google KMS](https://cloud.google.com/kms). We strongly recommended this ways to encrypt/decrypt your configuration,
+ but i you want a no vendor locking option or you haven`t the resources to use this methods, we create a way to encrypt
+ and decrypt your variables.
+ 
+## 1. Generate a key
+PyMS have a command line option to create a key file. this key is created with [AES](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
```

Comment:
Not sure about this, but I believe that when referencing a third person (or entity), the verb `have` has to mutate to `has`.
```suggestion
PyMS has a command line option to create a key file. this key is created with [AES](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
```